### PR TITLE
Update pages.md

### PR DESCRIPTION
**Pull request by jayliew**
Updated Credentials Sign-In code example to indicate how to use `getServerSideProps` but still also showing the older `getInitialProps` example

**What**:

**Why**:

**How**:

**Checklist**:

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

### CODE REVIEW

File: www/docs/configuration/pages.md [hunk 1]
```diff
@@ -106,11 +106,22 @@ export default function SignIn({ csrfToken }) {
   )
 }
 
+// This is the recommended way for Next.js 9.3 or newer
+export async function getServerSideProps(context){
+  const csrfToken = await getCsrfToken(context)
+  return {
+    props: { csrfToken }
+  }
+}
```
Comment (balazsorban44) [inline www/docs/configuration/pages.md:115]:
```suggestion
// This is the recommended way for Next.js 9.3 or newer
export async function getServerSideProps(context){
  return {
    props: {
      csrfToken: await getCsrfToken(context)
    }
  }
}
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thank you! This is appreciated. Could you please do the same at other occurrences of `getInitialProps` as well?

**Comment by jayliew** [comment]
> Thank you! This is appreciated. Could you please do the same at other occurrences of `getInitialProps` as well?

I'll make another pull request after this one gets merged.

**Comment by balazsorban44** [comment]
@jayliew couldnyoi please do the same for the other `getInitialProps` references as well?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
could you please incorporate #1596?

**Comment by jayliew** [comment]
@balazsorban44 How do I edit an existing PR for further modification?

**Comment by balazsorban44** [comment]
You check out the branch you worked on locally (`patch-1` in your case), and any commits pushed to it will be reflected in this PR, automatically!

**Review by balazsorban44 — APPROVED** [review]
Thank you! 🙂

**Pull request merged**